### ISSUE TITLE
[BUG] I get warnings when extracting oiio-2.4.10.0.zip on Windows via CMake

### ISSUE THREAD

**Issue by bebuch** [OpenImageIO/oiio#3811]
When extracting the [ZIP archive](https://codeload.github.com/OpenImageIO/oiio/zip/refs/tags/v2.4.10.0) on Windows (11) via cmake I get the warnings:

```
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim/ref/out2-freetype2.7.tif" -> "out-freetype2.7.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim/ref/out2-oldfreetype.tif" -> "out-oldfreetype.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim/ref/out2.tif" -> "out.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out2-freetype2.7.tif" -> "out-freetype2.7.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out2-oldfreetype.tif" -> "out-oldfreetype.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out2.tif" -> "out.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out3-freetype2.7.tif" -> "out-freetype2.7.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out3-oldfreetype.tif" -> "out-oldfreetype.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out3.tif" -> "out.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out4-freetype2.7.tif" -> "out-freetype2.7.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out4-oldfreetype.tif" -> "out-oldfreetype.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out4.tif" -> "out.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out5-freetype2.7.tif" -> "out-freetype2.7.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out5-oldfreetype.tif" -> "out-oldfreetype.tif".
cmake -E tar: warning: skipping symbolic link "oiio-2.4.10.0/testsuite/texture-udim2/ref/out5.tif" -> "out.tif".
```

Looks like the archive uses unit symbolic links which don't exist on Windows.

**Comment by lgritz** [comment]
I'm not sure this is an OIIO bug or anything we can directly fix. It's GitHub itself, not us, who provides the link that gives you a packed up zip file of the repo. And it's cmake, not us, who has the bug (if it's a bug) that can't deal with links when you do `cmake -E tar`.

Just a few guesses about where I'd go next if I were you:
* Which cmake version are you using? Is there a newer one that maybe has this fixed?
* Must you use `cmake -E tar`, or is there some other unzipping program that perhaps has no problem with these links?
* Must you download a zip file like this, or can you `git clone --depth 1 -b v2.4.10.0 ...` to (probably just as efficiently) get the code as a shallow git repo? (Shallow, because of --depth 1, means it won't waste time downloading the project's history or any branches other than the one you ask for.)

Does somebody with more Windows experience have a better suggestion? I don't recall ever seeing this problem reported before.

**Comment by lgritz** [comment]
I see. I think there are very few places in the repo that we purposely have symbolic links, maybe this is the only one. The reason it's there is that in that test, there are several individual test case reference images that are expected to be identical, so we were simply trying not to needlessly replicate the files taking up full space. I think if this is causing trouble, it would be ok to just replace those links with copies of the files. I will propose a patch that does this.

**Comment by lgritz** [comment]
Fix proposed in #3818
